docs/shared/asm-ingress-gateway/gateway.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: asm-ingressgateway
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: asm-ingressgateway
+spec:
+  type: LoadBalancer
+  selector:
+    asm: ingressgateway
+  ports:
+  - port: 80
+    name: http
+    targetPort: 8080
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: asm-ingressgateway
+spec:
+  replicas: 3
```

Comment:
**Question:** Is deploying 3 replicas (of the Anthos ingress gateway Pod) by default overkill? Or is this just a best practice?

Reply:
this is preemptive when we also show hpa! If you want i can take it out for now